packages/grafana-toolkit/src/cli/tasks/plugin.ci.ts

```diff
@@ -66,44 +70,18 @@ const buildPluginRunner: TaskRunner<PluginCIOptions> = async ({ finish, maxJestW
 
 export const ciBuildPluginTask = new Task<PluginCIOptions>('Build Plugin', buildPluginRunner);
 
-/**
- * 2. Build Docs
- *
- *  Take /docs/* and format it into /ci/docs/HTML site
- *
- */
-const buildPluginDocsRunner: TaskRunner<PluginCIOptions> = async () => {
-  const docsSrc = path.resolve(process.cwd(), 'docs');
-  if (!fs.existsSync(docsSrc)) {
-    console.log('No docs src');
-    return;
-  }
-
-  const start = Date.now();
-  const workDir = getJobFolder();
-  await execa('rimraf', [workDir]);
-  fs.mkdirSync(workDir);
-
-  const docsDest = path.resolve(process.cwd(), 'ci', 'docs');
-  fs.mkdirSync(docsDest);
-
-  const exe = await execa('cp', ['-rv', docsSrc + '/.', docsDest]);
-  console.log(exe.stdout);
-
-  fs.writeFileSync(path.resolve(docsDest, 'index.html'), `TODO... actually build docs`, { encoding: 'utf-8' });
```

Comment:
this never did anything... so should be removed anyway if any CI steps actually use it